com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -101,6 +101,7 @@ The version number for this package has increased due to a version update of a r
 - Exposed Parameter placement can be moved after sanitize
 - Fix rendering artifacts on some mobile devices [Case 1149057](https://issuetracker.unity3d.com/product/unity/issues/guid/1149057/)
 - Fix compilation failure on OpenGLES [Case 1348666](https://issuetracker.unity3d.com/product/unity/issues/guid/1348666/)
+- Don't open an empty VFX Graph Editor when assigning a VFX Asset to a Visual Effect GameObject from the inspector (https://issuetracker.unity3d.com/issues/vfx-assigning-an-empty-vfx-asset-to-vfx-game-object-results-in-a-duplicate)
```

Comment:
The formatting is wrong, it should be: [Case 1347399](https://issuetracker.unity3d.com/product/unity/issues/guid/1347399/)

```suggestion
- Don't open an empty VFX Graph Editor when assigning a VFX Asset to a Visual Effect GameObject from the inspector [Case 1347399](https://issuetracker.unity3d.com/product/unity/issues/guid/1347399/)
```